pkg/environment/local_execution_environment.go

```diff
@@ -66,9 +67,12 @@ func (e *localExecutionEnvironment) Run(ctx context.Context, request *runner.Run
 	if len(request.Arguments) < 1 {
 		return nil, status.Error(codes.InvalidArgument, "Insufficient number of command arguments")
 	}
-	cmd := exec.CommandContext(ctx, request.Arguments[0], request.Arguments[1:]...)
-	// TODO(edsch): Convert workingDirectory to use platform
-	// specific path delimiter.
+	executable := request.Arguments[0]
+	if runtime.GOOS == "windows" {
+		// Forward slash in the executable on Windows is interpreted as command argument. Convert to backslash.
```

Comment:
So this means that execute requests always use executable names containing forward slashes, even when targeting Windows? Ouch! Do we know whether this is something intentional on Bazel's side? Is there some existing option to switch the path separator convention here?

General nit: instead of conditionalizing this on Windows, what about doing this always? That keeps the code compact. The same likely holds for the working directory, right?

Reply:
> So this means that execute requests always use executable names containing forward slashes, even when targeting Windows? Ouch! Do we know whether this is something intentional on Bazel's side? Is there some existing option to switch the path separator convention here?

I haven't looked into what Bazel is meant to do and cannot find any documentation during a quick search. [`actions.run`](https://docs.bazel.build/versions/master/skylark/lib/actions.html#run) has a separate `executable` argument, so that might be part of it. Most use cases copes with forward slash on Windows, but apparently not everywhere. Probably need to do a proper check with a C api call to see if the problem is Bazel or Go not interacting correctly with `CreateProcess()`.

> General nit: instead of conditionalizing this on Windows, what about doing this always? That keeps the code compact. The same likely holds for the working directory, right?

Will fix the conditionalizing. Working directory does not need any handling, that one works.